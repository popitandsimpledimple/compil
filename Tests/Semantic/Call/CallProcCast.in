procedure f(i: Double; c: Double);
begin end;

begin
f(1.1, 1.1);
end.