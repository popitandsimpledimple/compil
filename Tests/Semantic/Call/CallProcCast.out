program
   decls
      procedure
         f
      params
         param
            i
            type
               double
         param
            c
            type
               double
         block
            compounds statements
   block
      compounds statements
         call
            f
               1.1
               1.1

 ───────────────────────────────────────────────────────────────────────────────────────────────────
 │ integer                   │ type                      │ integer                                 │
 │ char                      │ type                      │ char                                    │
 │ string                    │ type                      │ string                                  │
 │ double                    │ type                      │ double                                  │
 │ boolean                   │ type                      │ boolean                                 │
 ───────────────────────────────────────────────────────────────────────────────────────────────────

 ───────────────────────────────────────────────────────────────────────────────────────────────────
 │ without name              │ program name              │ no return type                          │
 │ f                         │ procedure                 │ no return type                          │
 ───────────────────────────────────────────────────────────────────────────────────────────────────

 ───────────────────────────────────────────────────────────────────────────────────────────────────
 │ i                         │ param                     │ double                                  │
 │ c                         │ param                     │ double                                  │
 ───────────────────────────────────────────────────────────────────────────────────────────────────
